pkg/types/vsphere/validation/platform_test.go

```diff
@@ -360,22 +360,6 @@ func TestValidatePlatform(t *testing.T) {
 			}(),
 			expectedError: `^test-path\.failureDomains\.zone: Required value: must specify zone tag value`,
 		},
-		{
-			name:     "forbidden load balancer field",
-			platform: validPlatform(),
-			config: &types.InstallConfig{
-				Platform: types.Platform{
-					VSphere: func() *vsphere.Platform {
-						p := validPlatform()
-						p.LoadBalancer = &configv1.VSpherePlatformLoadBalancer{
-							Type: configv1.LoadBalancerTypeOpenShiftManagedDefault,
-						}
-						return p
-					}(),
-				},
-			},
-			expectedError: `^test-path\.loadBalancer: Forbidden: load balancer is not supported in this feature set`,
-		},
```

Comment:
Similar test for baremetal (https://github.com/openshift/installer/blob/master/pkg/types/baremetal/validation/platform_test.go#L184-L190) needs to be removed too.

Reply:
Of course, somehow didn't pay enough attention